modules/core/03-connection/keeper/verify.go

```diff
@@ -159,9 +161,17 @@ func (k Keeper) VerifyPacketCommitment(
 		return sdkerrors.Wrapf(clienttypes.ErrClientNotActive, "client (%s) status is %s", clientID, status)
 	}
 
+	// calculate minimum block delay by dividing time delay period
+	// by the maximum expected time per block. Round up the block delay.
+	timeDelay := connection.GetDelayPeriod()
+	maxTimePerBlock := k.GetMaxTimePerBlock(ctx)
+	blockDelay := uint64(math.Ceil(float64(timeDelay) / float64(maxTimePerBlock)))
+
+	// verify that block delay has passed
```

Comment:
```suggestion
```
? seems misleading since the `VerifyPacketCommitment` is doing much more than just the block delay